[WIP] RHDEVDOCS 3910 and RHDEVDOCS-3902- Service binding discoverability and Topology usability improvements (#47760)
openshift/openshift-docs: modules/odc-verifying-the-status-of-your-service-binding-from-the-topology-view.adoc

The *Developer* perspective helps you verify the status of your service binding through the *Topology* view.

.Procedure

. If a service binding was successful, click the binding connector. A side panel appears displaying the *Connected* status under the *Details* tab. 
+
Optionally, you can view the *Connected* status on the following pages from the *Developer* perspective: 
+
** The *ServiceBindings* page.
** The *ServiceBinding details* page. In addition, the page title displays a *Connected* badge. 
. If a service binding was unsuccessful, the binding connector shows a red arrowhead and a red cross in the middle of the connection. Click this connector to view the *Error* status in the side panel under the *Details* tab. Optionally, click the *Error* status to view specific information about the underlying problem.
+
You can also view the *Error* status and a tooltip on the following pages from the *Developer* perspective:
+
** The *ServiceBindings* page.
** The *ServiceBinding details* page. In addition, the page title displays an *Error* badge.

[NOTE]
====
In the *ServiceBindings* page, use the *Filter* dropdown to list the service bindings based on their status.

> ====

Author: Done.

:_content-type: PROCEDURE
[id="odc-verifying-the-status-of-your-service-binding-from-the-topology-view_{context}"]
== Verifying the status of your service binding from the Topology view

The *Developer* perspective helps you verify the status of your service binding through the *Topology* view.

.Procedure
To verify the status of your service binding, consider the following approaches:

* Successful service binding
+

> When a service binding is successful, click on the binding connector. A side panel appears displaying the *Connected* status under the *Details* tab. 

Reviewer: Please apply the following IBM Style Guide guideline by replacing the 7 instances of "click on" in this PR with "click":

"Avoid using a phrasal verb (a verb plus a preposition) if the verb alone provides the same meaning." "click on > click"

Author: Done.

> To verify the status of your service binding, consider the following approaches:
> 
> * Successful service binding
> +
> When a service binding is successful, click on the binding connector. A side panel appears displaying the *Connected* status under the *Details* tab. 
> +
> Optionally, you can view the *Connected* status on the following pages from the *Developer* perspective: 
> +
> ** The *ServiceBindings* page.
> ** The *ServiceBinding details* page. In addition, the page title displays a *Connected* badge. 
> + 
> * Unsuccessful service binding
> +
> When a service binding is unsuccessful, the binding connector shows a red arrowhead and a red cross in the middle of the connection. Click on this connector to view the *Error* status in the side panel under the *Details* tab. Optionally, click on the *Error* status to view specific information about the underlying problem.
> +
> You can also view the *Error* status and a tooltip on the following pages from the *Developer* perspective:
> +
> ** The *ServiceBindings* page.
> ** The *ServiceBinding details* page. In addition, the page title displays an *Error* badge.

Reviewer: ```suggestion

. If a service binding was successful, click the binding connector. A side panel appears displaying the *Connected* status under the *Details* tab. 
+
Optionally, you can view the *Connected* status on the following pages from the *Developer* perspective: 
+
** The *ServiceBindings* page.
** The *ServiceBinding details* page. In addition, the page title displays a *Connected* badge. 

. If a service binding was unsuccessful, the binding connector shows a red arrowhead and a red cross in the middle of the connection. Click this connector to view the *Error* status in the side panel under the *Details* tab. Optionally, click on the *Error* status to view specific information about the underlying problem.
+
You can also view the *Error* status and a tooltip on the following pages from the *Developer* perspective:
+
** The *ServiceBindings* page.
** The *ServiceBinding details* page. In addition, the page title displays an *Error* badge.
```

> To verify the status of your service binding, consider the following approaches:
> 
> * Successful service binding
> +
> When a service binding is successful, click on the binding connector. A side panel appears displaying the *Connected* status under the *Details* tab. 
> +
> Optionally, you can view the *Connected* status on the following pages from the *Developer* perspective: 
> +
> ** The *ServiceBindings* page.
> ** The *ServiceBinding details* page. In addition, the page title displays a *Connected* badge. 
> + 
> * Unsuccessful service binding
> +
> When a service binding is unsuccessful, the binding connector shows a red arrowhead and a red cross in the middle of the connection. Click on this connector to view the *Error* status in the side panel under the *Details* tab. Optionally, click on the *Error* status to view specific information about the underlying problem.
> +
> You can also view the *Error* status and a tooltip on the following pages from the *Developer* perspective:
> +
> ** The *ServiceBindings* page.
> ** The *ServiceBinding details* page. In addition, the page title displays an *Error* badge.

Reviewer: ```suggestion

. If a service binding was successful, click the binding connector. A side panel appears displaying the *Connected* status under the *Details* tab. 
+
Optionally, you can view the *Connected* status on the following pages from the *Developer* perspective: 
+
** The *ServiceBindings* page.
** The *ServiceBinding details* page. In addition, the page title displays a *Connected* badge. 

. If a service binding was unsuccessful, the binding connector shows a red arrowhead and a red cross in the middle of the connection. Click this connector to view the *Error* status in the side panel under the *Details* tab. Optionally, click on the *Error* status to view specific information about the underlying problem.
+
You can also view the *Error* status and a tooltip on the following pages from the *Developer* perspective:
+
** The *ServiceBindings* page.
** The *ServiceBinding details* page. In addition, the page title displays an *Error* badge.
```

Author: Done.
